css/style.css

```diff
@@ -0,0 +1,250 @@
+*{
+  margin: 0;
+  padding: 0git;
+}
+
+.contact { 
+  width: 1500px;
+  height: 866px;
+  background: url(../image/2ce-bakg.png) no-repeat;
+  margin: 0 auto;
+}
+
+/*headr*/
+.contact-header { 
+  padding-top: 101px;
+}
+
+.contact-header__title { 
+  color: #262626;
+  font-family: "Raleway";
+  font-size: 90px;
+  font-weight: 700;
+  font-style: normal;
```

Comment:
`font-style: normal;` - is the default value. No need to write default CSS-rules. Remove all other rules that are default